cards/packs.go

```diff
@@ -0,0 +1,12 @@
+package cards
+
+type Pack struct {
+	Rare Card
+	Uncommons [3]Card
+	Commons [10]Card
+	Foil Card
```

Comment:
I think this might need to be a `*Card` as some packs didn't have Foils.  I think this is also true of Uncommons and Commons.  

Also, don't some sets have different numbers of Commons and Uncommons?  I think we might just want to simplify this a bit and model it so that a `Pack` just has an arbitrary amount of `Card`, like so:

```go
type Pack struct {
    Cards []*Card
}
```

And then we can have the rarity live on the Card itself:

```go
type CardRarity int

const(
    Common CardRarity = 0
    Uncommon CardRarity = 1
    Rare            CardRarity = 2
    Mythic         Card Rarity = 3
)

type Card struct {
   Rarity CardRarity
   Foil     bool
}
```